### PR TITLE
DOCS-6228 ORGANIZATION_NAME to Domain_Name

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

**Review by Mulesoft-Mysti — APPROVED** [review]
Technical issue resolved, thank you!

**Comment by wwarner-salesforce** [comment]
Certainly. Thank you!

File: modules/ROOT/pages/to-share-api-asset-to-portal.adoc [hunk 1]
```diff
@@ -6,7 +6,9 @@ endif::[]
 
 You can use your organization's public portal to share API assets publicly. Developers inside and outside your organization can access the assets on the public portal.
 
-A public portal is visible to anyone who has the URL of the portal. The URL for a public portal starts with `+https://anypoint.mulesoft.com/exchange/portals/ORGANIZATION_NAME/+`.
+A public portal is visible to anyone who has the URL of the portal. The URL for a public portal starts with `+https://anypoint.mulesoft.com/exchange/portals/Domain_Name/+`.
+
+Replace `Domain_Name` with the domain name shown in Anypoint Access Management on the xref:access-management::organization.adoc#organization-page[*Organization* page].
```
Comment (kbostrom) [inline modules/ROOT/pages/to-share-api-asset-to-portal.adoc]:
```suggestion
Replace `Domain_Name` with the domain name shown in Anypoint access management on the xref:access-management::organization.adoc#organization-page[*Organization* page].
```

Comment (kbostrom) [reply modules/ROOT/pages/to-share-api-asset-to-portal.adoc]: Yeah...I know.... But it really is lowercase. A lot of people aren't happy with that....

Comment (kbostrom) [reply modules/ROOT/pages/to-share-api-asset-to-portal.adoc]: Also, just doublechecking: I thought that HTTPS was secure, which implies to me (not technical) that public is at odds with that. I guess we mean "public" in the open exchange sense?

**Review by kbostrom — APPROVED** [review]
Done!

File: modules/ROOT/pages/to-share-api-asset-to-portal.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-share-api-asset-to-portal.adoc]: Oops! I simply didn't realize "Anypoint access management" is lowercase. That is not a problem at all for me, and I will try to remember from now on.

Interestingly HTTPS is secure in a slightly different way, and is used on most public sites such as google.com. Anyone with a standard web browser can go to google.com, establish a secure connection, and the search query they send to Google, and the search results Google sends back, cannot be eavesdropped on or tampered with by a third party, even though the user is still just some anonymous member of the public and Google does not know who they are or require them to log in. It's amazing technology when you think about it, even though most people use it all the time without knowing how it works. The Exchange public portal uses HTTPS the same way.

**Pull request merged**